statsmodels/stats/tests/test_diagnostic.py

```diff
@@ -287,6 +288,11 @@ def test_acorr_breusch_godfrey_multidim(self):
         with pytest.raises(ValueError, match='Model resid must be a 1d array'):
             smsdia.acorr_breusch_godfrey(res)
 
+    def test_acorr_breusch_godfrey_exogs(self):
+        data = sunspots.load_pandas().data['SUNACTIVITY']
+        res = ARMA(data, (1,0)).fit(disp=False, trend = 'nc')
+        smsdia.acorr_breusch_godfrey(res, nlags = 1)
```

Comment:
pep- 8 errors, no space around `=` on keyword arguments
previous line space after `,`

Reply:
I tried to fix in a new the syntax issues in a new commit in this branch. May you check?